src/components/DictionaryLinks/DictionaryLink.js

```diff
@@ -17,7 +17,7 @@ function DictionaryLink({ linkProperties }) {
       >
         {getIcon(
           linkProperties?.iconId,
-          'inline-flex fill-current w-6 xl:w-8 mr-2 xl:mr-5',
+          'btn-tertiary btn-md-icon inline-flex fill-current mr-2 xl:mr-5',
```

Comment:
Shouldn't this btn class be on the Link rather than the icon? Also, the icon styling on buttons with text should be automatic so no classes are needed on this icon.

Reply:
@Cara-Barter - @bronson-c set it up for us so that the styling is applied automatically to icons that are children of `btn-md` and other sizes. So in this case you should only need to apply `btn-tertiary btn-md` (or whatever size is appropriate for this use case) to the link/button itself. Let me know if you want to take a look together :)